modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -10,34 +10,44 @@ Run the cluster latency tests to validate node tuning for your Cloud-native Netw
+When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. Depending on your local OS and SELinux configuration, you might also experience issues running these commands from your home directory. To make the `podman` commands work, run the commands from a folder that is not your home directory, and append `:Z` to the volumes creation. For example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
 ====
 
 .Procedure
 
 . Open a shell prompt in the directory containing the `kubeconfig` file.
 +
-You provide the test image with a `kubeconfig` file in current directory and its related `$KUBECONFIG` environment variable, mounted through a volume. This allows the running container to use the `kubeconfig` file from inside the container.
-
-. Run the latency tests by entering the following command:
+You provide the test image with a `kubeconfig` file in current directory and its related `$KUBECONFIG` environment variable, mounted through a volume. This allows the running container to use the `kubeconfig` file from inside the container. 
++
+[NOTE]
+====
+In the following command, your local `kubeconfig` is mounted to kubeconfig/kubeconfig in the cnf-tests container, which allows access to the cluster.
+====
++
+. To run the latency tests, run the following command, substituting variable values as appropriate:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e LATENCY_TEST_RUNTIME=<time_in_seconds>\
--e MAXIMUM_LATENCY=<time_in_microseconds> \
+-e LATENCY_TEST_RUNTIME=300\
+-e MAXIMUM_LATENCY=20 \
 registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh \
 --ginkgo.v --ginkgo.timeout="24h"
 ----
-
-. Optional: Append `--ginkgo.dryRun` flag to run the latency tests in dry-run mode. This is useful for checking what commands the tests run.
++
+The latency tests run for 5 minutes (300 seconds). The test runs successfully when the maximum observed latency is lower than MAXIMUM_LATENCY (20 μs).
++
+If the results exceed the latency threshold, the test fails.
++
+. Optional: Append `--ginkgo.dry-run` flag to run the latency tests in dry-run mode. This is useful for checking what commands the tests run.
 
 . Optional: Append `--ginkgo.v` flag to run the tests with increased verbosity.
 
 . Optional: Append `--ginkgo.timeout="24h"` flag to ensure the Ginkgo 2.0 test suite does not timeout before the latency tests complete.
 +
 [IMPORTANT]
 ====
-The default runtime for each test is 300 seconds.
-For valid latency test results, run the tests for at least 12 hours by updating the `LATENCY_TEST_RUNTIME` variable.
+For valid results, the test should run for at least 12 hours.
```

Comment:
The info that says the default runtime is 300 is useful? Was there a reason for removing?

Reply:
Thanks @rohennes for your feedback. I removed the 300 to match 12 hours or (43200 seconds). The individual tests have LATENCY_TEST_RUNTIME=600.